cmd/nanotube/listen.go

```diff
@@ -74,12 +73,14 @@ loop:
 }
 
 func listenUDP(cfg *conf.Main, queue chan string, stop <-chan struct{}, lg *zap.Logger, ms *metrics.Prom) {
+	// TODO (grzkv) What should we do if connection drops?
```

Comment:
Crash with error message.
Common use case would be when there is another instance running&listening on the same port. We don't want to have the new instance hanging around if it fails to Listen(). 
Better to crash and let systemd/other system restart.

Reply:
This todo is more about "what to do if the connection drops in the middle of transmission".

Reply:
Ok. This comment belongs two lines below, will move it.